docs/source/scancode-toolkit/tutorials/how_to_add_a_post_scan_plugin.rst

```diff
@@ -0,0 +1,290 @@
+.. _how_to_add_post_scan_plugin:
+
+Add A Post-Scan Plugin
+======================
+
+Automatic vs. Manual Installation
+---------------------------------
+
+Automatic
```

Comment:
May be it would be better to call this built-in and optional rather than automatic and manual?

Reply:
Also the tutorial should be focused on optional plugins IMHO and leave aside the builtin ones as another topic

Reply:
@pombredanne Here I just replicated this [wiki](https://github.com/nexB/scancode-toolkit/wiki/How-To:-Add-a-post-scan-plugin) by @johnmhoran, i.e. this isn't the plugin docs that I'm writing on my own. Should I still go on and make these requested changes?

Reply:
yes, always feel free to update the content as you so that this is better and improved.

Reply:
Sure! On it.